src/Monitor/Monitor/help/Set-AzDiagnosticSetting.md

```diff
@@ -349,7 +349,7 @@ Accept wildcard characters: False
 ```
 
 ### -RetentionInDays
-Specifies the retention policy, in days.
+Specifies the retention policy, in days. Note that this only applies when saving data to a Storage account.
```

Comment:
Please update the help message of parameter as well. https://github.com/Azure/azure-powershell/blob/a8e2cee22d4efec03cd94837cb1c0f10539ce056/src/Monitor/Monitor/Diagnostics/SetAzureRmDiagnosticSettingCommand.cs#L130

Reply:
Good call, thanks - done!